src/python/bfiocpp/tswriter.py

```diff
@@ -5,18 +5,31 @@
 class TSWriter:
 
     def __init__(
-        self, file_name: str, image_shape: list, chunk_shape: list, dtype: np.dtype
+        self,
+        file_name: str,
+        image_shape: list,
+        chunk_shape: list,
+        dtype: np.dtype,
+        dimension_order: str,
     ):
         """Initialize tensorstore Zarr writer
 
         file_name: Path to write file to
         """
 
         self._image_writer: TsWriterCPP = TsWriterCPP(
-            file_name, image_shape, chunk_shape, str(dtype)
+            file_name, image_shape, chunk_shape, str(dtype), dimension_order
         )
 
-    def write_image_data(self, image_data: np.ndarray):
+    def write_image_data(
+        self,
+        image_data: np.ndarray,
+        rows: int,
+        cols: int,
+        layers: int,
+        channels: int,
+        tsteps: int,
+    ):
```

Comment:
This interface should support 2D-5D images. We should not have a restriction of only 5D image